deploy/backup.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: ps.percona.com/v1alpha1
+kind: PerconaServerMySQLBackup
+metadata:
+  name: backup5
```

Comment:
I guess you committed your test files, should be `backup1`

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/77/commits/b772f58a7b9d2e1df3ec49632b9b66ba6e80d9f9